### ISSUE TITLE
name 'init_empty_weights' is not defined

### ISSUE THREAD

**Issue by lillian521** [databrickslabs/dolly#45]
when I load the model with below code in the instruction, it reported the error ’name 'init_empty_weights' is not defined‘， please kindly advise how to fix, thanks a lot.
input code:
import numpy as np
from transformers import (
    AutoModelForCausalLM,
    AutoTokenizer,
    PreTrainedModel,
    PreTrainedTokenizer
)

tokenizer = AutoTokenizer.from_pretrained("databricks/dolly-v1-6b", padding_side="left")
model = AutoModelForCausalLM.from_pretrained("databricks/dolly-v1-6b", device_map="auto", trust_remote_code=True)

**Comment by srowen** [comment]
Where does the error come from? are you installing requirements.txt? that method comes from accelerate

**Comment by wvdm1217** [comment]
I had a similar issue and resolved it using `pip install accelerate` and reloading the notebook kernel I was using.

**Comment by srowen** [comment]
Right, there is a requirements.txt here that the provided script installs. If you're changing the training process and using it differently you still need these things
